src/utmParamsFromReferrer.js

```diff
@@ -5,6 +5,7 @@ import { parse as parseQuery } from './queryString'
 const engines = [
   { regex: /www.google./, query: 'q', name: 'google' },
   { regex: /yandex..*\/.*\?.*text=/, query: 'text', name: 'yandex' },
+  { regex: /^https?:\/\/(www\.)?yandex\.([a-z]{2,3})\/?$/, query: 'text', name: 'yandex' },
```

Comment:
`$` on end of regex means end of string.
So, it's okay that this regex **will not** match `http://yandex.ru/search`?

Also `^` means that start of string, so only url started from `http` will match.

Reply:
yes, only .ru or .ru/ ended!

Reply:
http/https started

Reply:
I think you should move this line before
```{ regex: /yandex..*\/.*\?.*text=/, query: 'text', name: 'yandex' },```

Reply:
https://monosnap.com/file/hIN99ogDJWKFjcYUUUapAM1fOoqaDb for example. More info you can see in team-segmentstream chat